website/src/.eslintrc.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I'm not sure if eslint works well with typescript. I would consider changing it to tslint

Reply:
Ok, sure. Changed to tslint